packages/geo-diary/src/reducers/auth.ts

```diff
@@ -11,14 +11,17 @@ import {
 import { RefreshParams, LoginSession, getSessionCookie } from '@reapit/cognito-auth'
 import { COOKIE_SESSION_KEY_GEO_DIARY } from '../constants/api'
 import { getMarketplaceGlobalsByKey } from '@reapit/elements'
+import config from '../../config.json'
```

Comment:
This one will cause error when async happen

Reply:
I think no, because we run `yarn fetch-config` before `yarn build`, and the `config.json` is always available. I cannot use `window.reapit.config.appEnv` because it's `async`